apps/ui/src/views/Proposal/Votes.vue

```diff
@@ -163,9 +163,10 @@ watch([sortBy, choiceFilter], () => {
           <UiStamp :id="vote.voter.id" :size="32" />
           <router-link
             :to="{
-              name: 'user',
+              name: 'space-user-statement',
               params: {
-                id: vote.voter.id
+                id: `${proposal.network}:${proposal.space.id}`,
+                user: vote.voter.id
```

Comment:
Similars to votes page on proposal, i think we should also redirect votes modal to `space-user-statement`, right now It redirect to `user`
<img width="856" alt="Untitled 6" src="https://github.com/user-attachments/assets/93b087f6-7759-4a85-a048-8175ce384b25">